db/migrate/20201015053916_create_frameworks_frameworks.rb

```diff
@@ -0,0 +1,11 @@
+class CreateFrameworksFrameworks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :frameworks_frameworks do |t|
```

Comment:
should be `framework_frameworks`

Reply:
in fact @nrankin not sure if these f*cked up migrations should be removed & re-done or migrated again?

Reply:
I'm just doing new ones as per rails best practice, having the incorrect names in older migrations won't cause any issues :)